yaml/csr.yaml.m4

```diff
@@ -2901,29 +2913,32 @@ ifelse(eval(I_BASE != 0), 1, [[[
       type: WARL
       msb: 0
       lsb: 0
+ifelse(eval(A_EXT == 0), 1, [[[
```

Comment:
A_EXT is no longer a single bit, but can be A_NONE, ZALRSC, A. Only if set to 'A' should the misa.A bit be set to 1. check all uses of A_EXT in this file.